docs/sonnet_with_special_thumbnail.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0"?>
+<OAI-PMH xmlns="http://www.openarchives.org/OAI/2.0/">
+  <ListRecords>
+    <record>
+      <metadata>
+        <mira_import xmlns:model="info:fedora/fedora-system:def/model#" xmlns:dc="http://purl.org/dc/terms/" xmlns:bibframe="http://bibframe.org/vocab/" xmlns:dc11="http://purl.org/dc/elements/1.1/" xmlns:ebucore="http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#" xmlns:premis="http://www.loc.gov/premis/rdf/v1#" xmlns:mads="http://www.loc.gov/mads/rdf/v1#" xmlns:marcrelators="http://id.loc.gov/vocabulary/relators/" xmlns:tufts="http://dl.tufts.edu/terms#" xmlns:scholarsphere="http://scholarsphere.psu.edu/ns#" xmlns:edm="http://www.europeana.eu/schemas/edm/" xmlns:foaf="http://xmlns.com/foaf/0.1/" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
+          <technicalmetadata>
```

Comment:
We said we'd ignore technical metadata on import, so I don't think we should include  lines 7-75.  In general if you're creating a new object or objects, hydra-file-characterization would supply the technical metadata.